tools/get_dicom.pl

```diff
@@ -0,0 +1,275 @@
+          is set to /tmp.
+-o      : basename of the final tar.gz file to produce, in the current directory (defaults to dicoms.tar.gz)
+
+=head1 DESCRIPTION
+
+This script first connects to the database to build the list of tarchives for which
+the patient names match the list of patterns specified as argument. The script will then examine 
+these tarchives and look for the MINC files whose scan types (acquisition protocol names) match 
+the list of patterns passed as argument. It then extracts the DICOM files associated to each
+MINC file and writes them in the extraction directory (see -d option), in a subdirectory with name
+
+<pscid>/<visit_label>/<acquisition_date>/<protocol>_<minc_index>
+
+where <minc_index> is the index number of the MINC file to which the DICOMs are associated.
+Finally, a tar is created which contains all the DICOM files that were extract
+
+=head2 Methods
+
+=cut
+
+use NeuroDB::DBI;
+use NeuroDB::ExitCodes;
+
+use File::Path;
+use File::Basename;
+use Getopt::Tabular;
+
+#-----------------------------------------------#
+# Triggers the deletion of the tmp extract dir  #
+# if something like CTRL-C is pressed during    #
+# execution of the script                       #
+#-----------------------------------------------#
+use sigtrap 'handler' => \&rmTmpExtractDir, 'normal-signals';
+
+my $patientNames;
+my $scanTypes;
+my $profile           = 'prod';
+my $tmpExtractBaseDir = '/tmp';
+my $outTarBasename    = 'dicoms';
+
+my @opt_table           = (
+    ["-p", "string", 1, \$profile,
+        "name of config file in ../dicom-archive/.loris_mri"],
+    ["-n", "string", 1,   \$patientNames, 
+        "comma-separated list of MySQL patterns for the patient name"],
+    ["-t", "string", 1,   \$scanTypes, 
+        "comma-separated list of MySQL patterns for the scan type"],
+    ["-d", "string", 1,   \$tmpExtractBaseDir, 
+        "base path of the temporary directory where files are extracted (defaults to /tmp)"],   
+    ["-o", "string", 1,   \$outTarBasename, 
+        "base path of the final .tar.gz file to produce (defaults to 'dicoms')"],   
+);
+
+my $Help = <<HELP;
+
+get_dicom.pl - extracts DICOM files for specific patient names/scan types
+
+This script first connects to the database to build the list of tarchives for which
+the patient names match the list of patterns specified as argument. The script will then examine 
+these tarchives and look for the MINC files whose scan types (acquisition protocol names) match 
+the list of patterns passed as argument. Once this is done, a tar file is built for each MINC 
+file containing the DICOM files associated with it. Each tar file is written in the extraction 
+directory(see -d option), in a subdirectory with name
+
+<pscid>/<visit_label>/<acquisition_date>/<protocol>_<minc_index>
+
+where <minc_index> is the index number of the MINC file for which the tar is produced.
+Finally, a tar is created which contains all the DICOM files that were extract
```

Comment:
a **.tar.gz file` is created ... that were **extracted**

Reply:
Fixed.